test/config/utility.h

```diff
@@ -219,6 +220,19 @@ class ConfigHelper {
   // Modifiers will be applied just before ports are modified in finalize
   void addConfigModifier(HttpModifierFunction function);
 
+  // Allows callers to easily modify the filter named 'name' from the first filter chain from the
```

Comment:
Thanks, this is great. Does it work to re-express the existing store/loadHttpConnectionManager in terms of the new methods you introduce?

Reply:
Yes, I have rewrote store/loadHttpConnectionManager using load/storeFilter.